omod/src/main/webapp/pages/deliveryNotes.gsp

```diff
@@ -523,11 +523,26 @@
                     Not within the hour
                 </label>
             </div>
-			
-            <div>
-                <label for="apgar-score">APGAR Score<span class="required">*</span></label>
-                <input type="text" id="apgar-score" name="concept.1504AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA" class="number numeric-range" value="" placeholder="APGAR Score">
-            </div>
+			<br>
+			<div>
+				<label for="birth-notification-number">APGAR Score at 1<span class="required">*</span></label>
+				<input type="number" name="" class="number numeric-range" value="" placeholder="score value">
+			</div>
+
+			<div>
+				<label for="birth-notification-number">APGAR Score at 5<span class="required">*</span></label>
```

Comment:
All these are NOT linked to any concepts? we can't record any information on such fields